test_fzz.py

```diff
@@ -5,7 +5,12 @@
     fzz = pyfzz()
     data = fzz.read_file('sample_filt')
     bars = fzz.compute_zigzag(data)
+    print(f'Bars from file {bars}')
+    # Write the result into a file.
+    
     fzz.write_file('sample_filt_pers', bars)
-    # bars = fzz.compute_zigzag([('i', [0]), ('i', [1]), ('i', [0, 1])])
 
+    # You can use the following code to test the function directly with a list of insertion and deletion operations.
+    
+    bars = fzz.compute_zigzag([('i', [0]), ('i', [1]), ('i', [0, 1])])
```

Comment:
Oh, I see, you added some example here. So I am guessting 'd' would mean deletion (in that case can you add some deletions to make it clearer -- this would be good for a sample program). Also, I didn't check, but have added in the readme about how to use the python interface?

Reply:
I added a 'd' in the sample program.